spreadsheets/sort_rank/water.csv

```diff
@@ -1,16 +1,16 @@
 kind,boundary,sort_rank
-river,,201
```

Comment:
What does the `-` mean here?

Reply:
It means `boundary` is not set. The empty match in `river,,201` translates to `kind == 'river' and boundary == ''`, which never matched (`boundary` is either `None` or `True`), so all water features were getting a `sort_rank` of 200.